python/smashbox/utilities/__init__.py

```diff
@@ -668,20 +668,24 @@ def check_groups(num_groups=None):
             fatal_check(result, 'Group %s not found' % group_name)
 
 
-def expect_modified(fn, md5):
+def expect_modified(fn, md5, comparedTo=''):
```

Comment:
yes, it's an inline comment, to make the error message in the log easier to unserstand.

Fixed it.